src/packages/@ncigdc/containers/explore/ExplorePage.js

```diff
@@ -130,19 +137,35 @@ class ExplorePageComponent extends React.Component {
               <CaseAggregations
                 aggregations={viewer.explore.cases.aggregations}
                 facets={viewer.explore.customCaseFacets}
+                maxFacetsPanelHeight={maxFacetsPanelHeight}
                 setAutocomplete={(value, onReadyStateChange) => relay.setVariables(
                   {
                     idAutocompleteCases: value,
                     runAutocompleteCases: !!value,
                   },
-                  onReadyStateChange
+                  onReadyStateChange,
                 )}
                 suggestions={get(viewer, 'autocomplete_cases.hits', [])}
                 />
             ),
             id: 'cases',
             text: 'Cases',
           },
+          {
+            component: (
+              <ClinicalAggregations
+                aggregations={viewer.explore.cases.aggregations}
+                caseFacets={viewer.caseFacets}
+                docType="cases"
+                facets={viewer.explore.customCaseFacets}
+                globalFilters={filters}
+                maxFacetsPanelHeight={maxFacetsPanelHeight}
+                relayVarName="exploreCaseCustomFacetFields"
+                />
+            ),
+            id: 'clinical',
+            text: 'Clinical',
+          },
           {
             component: (
               <GeneAggregations relay={relay} />
```

Comment:
not sure if GeneAggregations needs to account for the panel height prop or not. we'll have to verify at some point.

Reply:
Is that something I've missed in this PR? or just a thought for something we should look at later?

Reply:
something for later...when i refactored Gene Aggs i don't remember seeing anything for the panel height so it might be fine as is. Gene Aggs don't have a lot of facets